### PR TITLE
js_task_fix_form_DOM: add tests

### PR DESCRIPTION
**Pull request by brainwashers**

### CODE REVIEW

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+describe('Fixed form', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
```
Comment (roman-mate) [inline cypress/integration/jsTaskFixFormDOM.spec.js:4]: https://docs.cypress.io/guides/references/best-practices.html#Setting-a-global-baseUrl

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  it('should have placeholder for first name', () => {
+    cy.get(':nth-child(1) > :nth-child(1) > input').should('have.attr', 'placeholder');
```
Comment (roman-mate) [inline cypress/integration/jsTaskFixFormDOM.spec.js:7]: is there a better css selector?

Comment (roman-mate) [reply cypress/integration/jsTaskFixFormDOM.spec.js:7]: same below.

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  it('should have placeholder for email', () => {
+    cy.get('form:nth-child(2) .field:nth-child(1) > input').should('have.attr', 'placeholder');
+  });
+
+  it('should have placeholder for password', () => {
+    cy.get('form:nth-child(2) .field:nth-child(2) > input').should('have.attr', 'placeholder');
+  });
+
+  it('should have label for first name', () => {
+    cy.get('form:nth-child(1) > :nth-child(1) > label').should('exist');
```
Comment (roman-mate) [inline cypress/integration/jsTaskFixFormDOM.spec.js:31]: need better css selectors.

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  });
+
+  it('should have label for email', () => {
+    cy.get('form:nth-child(2) > :nth-child(1) > label').should('exist');
+  });
+
+  it('should have label for password', () => {
+    cy.get('form:nth-child(2) :nth-child(2) label').should('exist');
+  });
+});
```
Comment (roman-mate) [inline cypress/integration/jsTaskFixFormDOM.spec.js:53]: missing empty line

**Review by roman-mate — CHANGES_REQUESTED** [review]

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+  it('should have placeholder for password in sign in form', () => {
+    cy.get('#sign-in-password').should('have.attr', 'placeholder');
+  });
+
+  it('should have label for first name in sign up form', () => {
+    cy.get('#sign-up-name').parent().children('label');
+  });
+
+  it('should have abel for last name in sign up form', () => {
```
Comment (roman-mate) [inline cypress/integration/jsTaskFixFormDOM.spec.js:34]:
```suggestion
  it('should have label for last name in sign up form', () => {
```

**Review by roman-mate — APPROVED** [review]

File: cypress/integration/jsTaskFixFormDOM.spec.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  it('should have placeholder for email in sign in form', () => {
+    cy.get('#sign-in-email').should('have.attr', 'placeholder');
+  });
+
+  it('should have placeholder for password in sign in form', () => {
+    cy.get('#sign-in-password').should('have.attr', 'placeholder');
+  });
+
+  it('should have label for first name in sign up form', () => {
+    cy.get('#sign-up-name').parent().children('label');
```
Comment (rhonyshniuk) [inline cypress/integration/jsTaskFixFormDOM.spec.js:31]: `cy.get('[for="sign-up-email"]')`

Change for all cases

**Pull request merged**